test/test-plugin/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "name": "test-plugin",
+  "version": "1.0.0",
+  "description": "Fake plugin for testing code-server's plugin API",
+  "engines": {
+    "code-server": "^3.6.0"
+  },
+  "main": "out/index.js",
```

Comment:
Any thoughts on how we should build this before running `yarn test`? What comes to mind for me is:

- Require you `cd` in and build it first (maybe with an entry in `CONTRIBUTING.md`)
- Do it automatically as part of the `yarn test` script
- Remove `main` and move index.ts to the root of the plugin so it works with both Typescript directly or with Javascript

Reply:
My bad, forgot to build this before `yarn test`.

The reason it has its own build process is that the top level `tsconfig.json` has its top level dir as `src`. But I'll try add it there instead if possible.